packages/p2p/src/pages/my-profile/my-profile-stats/__tests__/my-profile-stats.spec.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import { screen, render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { isMobile } from '@deriv/shared';
```

Comment:
isMobile => is_mobile

Reply:
i left the isMobile for some of the test files because of `MobileWrapper` being used in the component :/ so i can't really change it until we change that component

Reply:
ah.. okay. 👍